doc/source/cookbook.rst

```diff
@@ -1354,3 +1354,37 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Common Excel Operations
+-----------------------
+
+Fill Handle
+***********
+
+Create a series of numbers following a set pattern in a certain set of cells.
+
+.. ipython:: python
+
+    df = pd.DataFrame({'AAA': [1]*8, 'BBB': list(range(0,8))}); df
+
+    # Fill numbers with difference 4 starting from 1 
+    # in rows 2 to 5 in column AAA.
+
+    df.iloc[2:(5+1)].AAA = [ x*4 + 2 for x in range(0, len(df.iloc[2:(5+1)])) ]
```

Comment:
I find this a bit to complex for a recipe. I'd prefer the case when in Excel you've got a `1` in the first cell, and you drag it creating a sequence. I think that case is more common, and the code will be much easier to read. What do you think?

Reply:
Dragging one down will just fill ones in excel. I'm trying to show that they can insert whatever series they want irrespective of how complex it is. But you're right I should change it to something simpler and just say that more can be done in text.

Reply:
hehe, let's say shift+dragging, so it's `1, 2, 3...`. :) May be you can show both examples, starting by the simplest. I just think showing the current code directly, will feels a bit too scary for an average pandas user.

Reply:
I agree. I'll change it.

Reply:
btw, I don't use excel that much myself, but may be Excel users could also appreciate a short example on search/replace equivalent?

Reply:
Yeah, maybe using apply()

I was thinking pivot tables, drop duplicates, formulae, adding a row or multiple rows, random data, vlookup. 

Excel makes some plots really easy, should that be included?

Reply:
That sounds good, yes, but then let's move this from the cookbook to a new documentation page "Comparison with Excel". You can find pages for Comparison with SQL, SAS and Stata already (check the end of the left sidebar here: http://pandas.pydata.org/pandas-docs/stable/)